src/lib/ch-engine/index.ts

```diff
@@ -0,0 +1,44 @@
+import { invoke } from '@tauri-apps/api/tauri';
+import type { CHResponse, ColumnDescriptor, Source } from './types';
+
+export async function init() {
+	await exec(`
+		CREATE DATABASE IF NOT EXISTS agx;
+		USE agx;
+		CREATE TABLE IF NOT EXISTS ethereum_mainnet_logs ENGINE = S3('https://data.agnostic.dev/ethereum-mainnet-pq/logs/*.parquet');
```

Comment:
We can omit this so AGX becomes more friendly for people who are not interested in blockchain data.